src/routes/event.js

```diff
@@ -1,15 +1,17 @@
-import { Router } from 'express';
-import asyncHandler from '../middlewares/asyncHandler';
-import eventController from '../controllers/event';
-import { auth, adminAuth } from '../middlewares/authorization';
+import { Router } from "express";
+import asyncHandler from "../middlewares/asyncHandler";
+import eventController from "../controllers/event";
+import { auth, adminAuth } from "../middlewares/authorization";
+import { getSignedUrl } from "../controllers/files";
 
 
 const eventRouter = new Router();
 
 eventRouter
-    .post('/', [auth, adminAuth], asyncHandler(eventController.postEvent))
-    .get('/', [auth, adminAuth], asyncHandler(eventController.getEvents))
-    .delete('/:id', [auth, adminAuth], asyncHandler(eventController.deleteEvent))
-    .put('/:id', [auth, adminAuth], asyncHandler(eventController.updateEvent))
+    .post("/", [auth, adminAuth], asyncHandler(eventController.postEvent))
+    .get("/signedUrl", [auth, adminAuth], asyncHandler(getSignedUrl))
+    .get("/", [auth, adminAuth], asyncHandler(eventController.getEvents))
+    .delete("/:id", [auth, adminAuth], asyncHandler(eventController.deleteEvent))
+    .put("/:id", [auth, adminAuth], asyncHandler(eventController.updateEvent))
```

Comment:
This route does not work, probably the `eventController.updateEvent()` function might not be working probably.

Reply:
This actually works as I conditioned.... I made it only to update caption because I thought for the event you really only need to update caption not name(title) of event.